src/App.vue

```diff
@@ -1,118 +1,70 @@
-        <FilterSearch :filters="filters" />
-        <v-btn
-          color="primary"
-          prepend-icon="mdi-pencil"
-          @click="showSnackbar('info', 'Custom button clicked')"
-        >
-          Modo edición
-        </v-btn>
-      </v-row>
-
-      <v-container class="fill-height">
-        <v-responsive class="flex justify-center overflow-visible">
-          <MButton
-            label="HoLi"
-            icon="mdi-heart"
-            icon-position="prepend-icon"
-            color="#020246"
-            rounded="xl"
-            size="small"
-          />
-        </v-responsive>
-      </v-container>
-
-      <v-container class="fill-height">
-          <FilterSearch :filters="filters" @search="handleSearch" @filter="handleFilters"/>
-          <KPIChartCard
-            title="Occupation rate of filling racks"
-            fontSize="20px"
-            height="200px"
-            :series="[90, 31, 80, 40, 51, 42, 109, 100]"
-            chartType="area"
-            backgroundColor="#f2b6d6"
-            titleColor="#8c0e21"
-            valueColor="#8c0e21"
-            valueFontSize="46px"
-            chartColor="#214270"
-            chartHeight="200px"
-            chartWidth="200px"
-          />
-          <KPIChartCard
-            title="Occupation rate of filling racks"
-            :value="50"
-            chartType="donut"
-            backgroundColor="#f2b6d6"
-            chartColor="#1b1c2e"
-            titleColor="#8c0e21"
-            valueColor="#8c0e21"
-            height="200px"
-            chartHeight="200px"
-            chartWidth="200px"
-            valueFontSize="46px"
-            fontSize="20px"
-          />
-      </v-container>
-    </v-main>
+    <MDataTable
+      style="height: 200px"
+      key="data-checks-table"
+      class="data-checks-table"
+      :items="filteredDataTable"
+      :headers="headers"
+      :showFooter="true"
+      :options="{ density: 'compact' }"
+      :prevText="'Anterior'"
+    />
   </v-app>
 </template>
 
 <script setup lang="ts">
-import FilterSearch from '@/components/FilterSearch/FilterSearch.vue'
-import MButton from '@/components/Button/MButton.vue'
-import KPIChartCard from '@/components/KPICharts/KPIChartCard.vue'
+import MDataTable from './components/MDataTable/MDataTable.vue'
 
+const filteredDataTable = [
+  { id: 1, name: 'Juan', manager: true },
+  { id: 2, name: 'Pedro', manager: true },
+  { id: 3, name: 'Maria', manager: true },
+  { id: 4, name: 'Jose', manager: true },
+  { id: 5, name: 'Luis', manager: true },
+  { id: 6, name: 'Ana', manager: false },
+  { id: 7, name: 'Rosa', manager: false },
+  { id: 8, name: 'Lorena', manager: false },
+  { id: 9, name: 'Luisa', manager: false },
+  { id: 10, name: 'Miguel', manager: false },
+  { id: 11, name: 'Carlos', manager: false },
+  { id: 12, name: 'Sofia', manager: false },
+  { id: 13, name: 'Sara', manager: false },
+  { id: 14, name: 'Sandra', manager: false },
+]
```

Comment:
Why do we have this data here?

Reply:
Yes, you are right. This is code just for testing and should not be uploaded